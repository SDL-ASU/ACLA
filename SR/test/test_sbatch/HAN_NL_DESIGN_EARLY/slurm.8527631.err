  0%|                                                     | 0/5 [00:00<?, ?it/s] 20%|█████████                                    | 1/5 [00:00<00:03,  1.02it/s] 40%|██████████████████                           | 2/5 [00:01<00:02,  1.45it/s] 60%|███████████████████████████                  | 3/5 [00:01<00:01,  1.73it/s] 80%|████████████████████████████████████         | 4/5 [00:02<00:00,  1.85it/s]100%|█████████████████████████████████████████████| 5/5 [00:02<00:00,  1.83it/s]
  0%|                                                    | 0/14 [00:00<?, ?it/s]  7%|███▏                                        | 1/14 [00:00<00:12,  1.02it/s] 14%|██████▎                                     | 2/14 [00:01<00:08,  1.39it/s] 21%|█████████▍                                  | 3/14 [00:01<00:07,  1.53it/s] 29%|████████████▌                               | 4/14 [00:02<00:06,  1.47it/s] 36%|███████████████▋                            | 5/14 [00:03<00:05,  1.51it/s] 43%|██████████████████▊                         | 6/14 [00:03<00:05,  1.53it/s] 50%|██████████████████████                      | 7/14 [00:04<00:04,  1.60it/s] 57%|█████████████████████████▏                  | 8/14 [00:04<00:03,  1.65it/s] 64%|████████████████████████████▎               | 9/14 [00:05<00:02,  1.67it/s] 71%|██████████████████████████████▋            | 10/14 [00:05<00:02,  1.72it/s] 79%|█████████████████████████████████▊         | 11/14 [00:06<00:01,  1.73it/s] 86%|████████████████████████████████████▊      | 12/14 [00:06<00:01,  1.72it/s] 93%|███████████████████████████████████████▉   | 13/14 [00:07<00:00,  1.73it/s]100%|███████████████████████████████████████████| 14/14 [00:08<00:00,  1.75it/s]
  0%|                                                   | 0/100 [00:00<?, ?it/s]  1%|▍                                          | 1/100 [00:00<01:14,  1.33it/s]  2%|▊                                          | 2/100 [00:01<00:58,  1.68it/s]  3%|█▎                                         | 3/100 [00:01<00:52,  1.85it/s]  4%|█▋                                         | 4/100 [00:02<00:49,  1.93it/s]  5%|██▏                                        | 5/100 [00:02<00:46,  2.02it/s]  6%|██▌                                        | 6/100 [00:02<00:45,  2.05it/s]  7%|███                                        | 7/100 [00:03<00:44,  2.11it/s]  8%|███▍                                       | 8/100 [00:03<00:44,  2.06it/s]  9%|███▊                                       | 9/100 [00:04<00:44,  2.05it/s] 10%|████▏                                     | 10/100 [00:04<00:43,  2.06it/s] 11%|████▌                                     | 11/100 [00:05<00:42,  2.08it/s] 12%|█████                                     | 12/100 [00:05<00:41,  2.10it/s] 13%|█████▍                                    | 13/100 [00:06<00:41,  2.12it/s] 14%|█████▉                                    | 14/100 [00:06<00:40,  2.14it/s] 15%|██████▎                                   | 15/100 [00:06<00:39,  2.15it/s] 16%|██████▋                                   | 16/100 [00:07<00:38,  2.17it/s] 17%|███████▏                                  | 17/100 [00:07<00:38,  2.15it/s] 18%|███████▌                                  | 18/100 [00:08<00:38,  2.16it/s] 19%|███████▉                                  | 19/100 [00:08<00:37,  2.16it/s] 20%|████████▍                                 | 20/100 [00:09<00:36,  2.16it/s] 21%|████████▊                                 | 21/100 [00:09<00:36,  2.18it/s] 22%|█████████▏                                | 22/100 [00:10<00:35,  2.19it/s] 23%|█████████▋                                | 23/100 [00:10<00:35,  2.20it/s] 24%|██████████                                | 24/100 [00:10<00:34,  2.20it/s] 25%|██████████▌                               | 25/100 [00:11<00:34,  2.17it/s] 26%|██████████▉                               | 26/100 [00:11<00:34,  2.17it/s] 27%|███████████▎                              | 27/100 [00:12<00:33,  2.17it/s] 28%|███████████▊                              | 28/100 [00:12<00:33,  2.17it/s] 29%|████████████▏                             | 29/100 [00:13<00:32,  2.17it/s] 30%|████████████▌                             | 30/100 [00:13<00:32,  2.17it/s] 31%|█████████████                             | 31/100 [00:14<00:31,  2.16it/s] 32%|█████████████▍                            | 32/100 [00:14<00:31,  2.16it/s] 33%|█████████████▊                            | 33/100 [00:15<00:31,  2.13it/s] 34%|██████████████▎                           | 34/100 [00:16<00:31,  2.12it/s] 35%|██████████████▋                           | 35/100 [00:16<00:30,  2.11it/s] 36%|███████████████                           | 36/100 [00:17<00:30,  2.11it/s] 37%|███████████████▌                          | 37/100 [00:17<00:29,  2.11it/s] 38%|███████████████▉                          | 38/100 [00:18<00:29,  2.10it/s] 39%|████████████████▍                         | 39/100 [00:18<00:29,  2.10it/s] 40%|████████████████▊                         | 40/100 [00:19<00:28,  2.08it/s] 41%|█████████████████▏                        | 41/100 [00:19<00:28,  2.08it/s] 42%|█████████████████▋                        | 42/100 [00:20<00:27,  2.08it/s] 43%|██████████████████                        | 43/100 [00:20<00:27,  2.07it/s] 44%|██████████████████▍                       | 44/100 [00:21<00:27,  2.07it/s] 45%|██████████████████▉                       | 45/100 [00:21<00:26,  2.07it/s] 46%|███████████████████▎                      | 46/100 [00:22<00:26,  2.07it/s] 47%|███████████████████▋                      | 47/100 [00:22<00:25,  2.06it/s] 48%|████████████████████▏                     | 48/100 [00:23<00:25,  2.06it/s] 49%|████████████████████▌                     | 49/100 [00:23<00:24,  2.06it/s] 50%|█████████████████████                     | 50/100 [00:24<00:24,  2.06it/s] 51%|█████████████████████▍                    | 51/100 [00:24<00:23,  2.04it/s] 52%|█████████████████████▊                    | 52/100 [00:25<00:23,  2.03it/s] 53%|██████████████████████▎                   | 53/100 [00:26<00:23,  2.03it/s] 54%|██████████████████████▋                   | 54/100 [00:26<00:22,  2.03it/s] 55%|███████████████████████                   | 55/100 [00:27<00:22,  2.03it/s] 56%|███████████████████████▌                  | 56/100 [00:27<00:21,  2.02it/s] 57%|███████████████████████▉                  | 57/100 [00:28<00:21,  2.02it/s] 58%|████████████████████████▎                 | 58/100 [00:28<00:20,  2.02it/s] 59%|████████████████████████▊                 | 59/100 [00:29<00:20,  2.02it/s] 60%|█████████████████████████▏                | 60/100 [00:29<00:19,  2.02it/s] 61%|█████████████████████████▌                | 61/100 [00:30<00:19,  2.02it/s] 62%|██████████████████████████                | 62/100 [00:30<00:18,  2.02it/s] 63%|██████████████████████████▍               | 63/100 [00:31<00:18,  2.02it/s] 64%|██████████████████████████▉               | 64/100 [00:31<00:17,  2.02it/s] 65%|███████████████████████████▎              | 65/100 [00:32<00:17,  2.02it/s] 66%|███████████████████████████▋              | 66/100 [00:32<00:16,  2.01it/s] 67%|████████████████████████████▏             | 67/100 [00:33<00:16,  2.02it/s] 68%|████████████████████████████▌             | 68/100 [00:33<00:15,  2.02it/s] 69%|████████████████████████████▉             | 69/100 [00:34<00:15,  2.01it/s] 70%|█████████████████████████████▍            | 70/100 [00:34<00:14,  2.01it/s] 71%|█████████████████████████████▊            | 71/100 [00:35<00:14,  2.01it/s] 72%|██████████████████████████████▏           | 72/100 [00:35<00:13,  2.01it/s] 73%|██████████████████████████████▋           | 73/100 [00:36<00:13,  2.01it/s] 74%|███████████████████████████████           | 74/100 [00:36<00:12,  2.01it/s] 75%|███████████████████████████████▌          | 75/100 [00:37<00:12,  2.01it/s] 76%|███████████████████████████████▉          | 76/100 [00:37<00:11,  2.01it/s] 77%|████████████████████████████████▎         | 77/100 [00:38<00:11,  2.01it/s] 78%|████████████████████████████████▊         | 78/100 [00:38<00:10,  2.01it/s] 79%|█████████████████████████████████▏        | 79/100 [00:39<00:10,  2.01it/s] 80%|█████████████████████████████████▌        | 80/100 [00:39<00:09,  2.01it/s] 81%|██████████████████████████████████        | 81/100 [00:40<00:09,  2.01it/s] 82%|██████████████████████████████████▍       | 82/100 [00:40<00:08,  2.01it/s] 83%|██████████████████████████████████▊       | 83/100 [00:41<00:08,  2.01it/s] 84%|███████████████████████████████████▎      | 84/100 [00:41<00:07,  2.01it/s] 85%|███████████████████████████████████▋      | 85/100 [00:42<00:07,  2.01it/s] 86%|████████████████████████████████████      | 86/100 [00:42<00:06,  2.01it/s] 87%|████████████████████████████████████▌     | 87/100 [00:43<00:06,  2.01it/s] 88%|████████████████████████████████████▉     | 88/100 [00:43<00:05,  2.01it/s] 89%|█████████████████████████████████████▍    | 89/100 [00:44<00:05,  2.01it/s] 90%|█████████████████████████████████████▊    | 90/100 [00:44<00:04,  2.00it/s] 91%|██████████████████████████████████████▏   | 91/100 [00:45<00:04,  2.00it/s] 92%|██████████████████████████████████████▋   | 92/100 [00:45<00:03,  2.00it/s] 93%|███████████████████████████████████████   | 93/100 [00:46<00:03,  1.99it/s] 94%|███████████████████████████████████████▍  | 94/100 [00:47<00:03,  1.99it/s] 95%|███████████████████████████████████████▉  | 95/100 [00:47<00:02,  1.99it/s] 96%|████████████████████████████████████████▎ | 96/100 [00:48<00:02,  2.00it/s] 97%|████████████████████████████████████████▋ | 97/100 [00:48<00:01,  2.00it/s] 98%|█████████████████████████████████████████▏| 98/100 [00:49<00:01,  2.00it/s] 99%|█████████████████████████████████████████▌| 99/100 [00:49<00:00,  2.00it/s]100%|█████████████████████████████████████████| 100/100 [00:49<00:00,  2.01it/s]
  0%|                                                   | 0/100 [00:00<?, ?it/s]  1%|▍                                          | 1/100 [00:01<02:06,  1.28s/it]  2%|▊                                          | 2/100 [00:02<02:11,  1.34s/it]  3%|█▎                                         | 3/100 [00:03<01:57,  1.21s/it]  4%|█▋                                         | 4/100 [00:04<01:52,  1.17s/it]  5%|██▏                                        | 5/100 [00:05<01:53,  1.20s/it]  6%|██▌                                        | 6/100 [00:07<01:50,  1.18s/it]  7%|███                                        | 7/100 [00:08<01:51,  1.20s/it]  8%|███▍                                       | 8/100 [00:09<01:47,  1.17s/it]  9%|███▊                                       | 9/100 [00:10<01:43,  1.14s/it] 10%|████▏                                     | 10/100 [00:11<01:43,  1.15s/it] 11%|████▌                                     | 11/100 [00:12<01:40,  1.13s/it] 12%|█████                                     | 12/100 [00:13<01:37,  1.10s/it] 13%|█████▍                                    | 13/100 [00:14<01:35,  1.09s/it] 14%|█████▉                                    | 14/100 [00:15<01:32,  1.08s/it] 15%|██████▎                                   | 15/100 [00:15<01:30,  1.06s/it] 16%|██████▋                                   | 16/100 [00:16<01:28,  1.06s/it] 17%|███████▏                                  | 17/100 [00:18<01:28,  1.06s/it] 18%|███████▌                                  | 18/100 [00:19<01:27,  1.06s/it] 19%|███████▉                                  | 19/100 [00:19<01:25,  1.05s/it] 20%|████████▍                                 | 20/100 [00:20<01:23,  1.04s/it] 21%|████████▊                                 | 21/100 [00:22<01:22,  1.05s/it] 22%|█████████▏                                | 22/100 [00:22<01:21,  1.04s/it] 23%|█████████▋                                | 23/100 [00:23<01:20,  1.04s/it] 24%|██████████                                | 24/100 [00:24<01:18,  1.04s/it] 25%|██████████▌                               | 25/100 [00:26<01:18,  1.04s/it] 26%|██████████▉                               | 26/100 [00:27<01:18,  1.06s/it] 27%|███████████▎                              | 27/100 [00:28<01:16,  1.05s/it] 28%|███████████▊                              | 28/100 [00:29<01:15,  1.05s/it] 29%|████████████▏                             | 29/100 [00:30<01:14,  1.05s/it] 30%|████████████▌                             | 30/100 [00:31<01:13,  1.05s/it] 31%|█████████████                             | 31/100 [00:32<01:12,  1.04s/it] 32%|█████████████▍                            | 32/100 [00:33<01:10,  1.04s/it] 33%|█████████████▊                            | 33/100 [00:34<01:09,  1.04s/it] 34%|██████████████▎                           | 34/100 [00:35<01:08,  1.04s/it] 35%|██████████████▋                           | 35/100 [00:36<01:07,  1.04s/it] 36%|███████████████                           | 36/100 [00:37<01:06,  1.03s/it] 37%|███████████████▌                          | 37/100 [00:38<01:04,  1.03s/it] 38%|███████████████▉                          | 38/100 [00:38<01:03,  1.02s/it] 39%|████████████████▍                         | 39/100 [00:39<01:02,  1.02s/it] 40%|████████████████▊                         | 40/100 [00:40<01:00,  1.02s/it] 41%|█████████████████▏                        | 41/100 [00:41<01:00,  1.02s/it] 42%|█████████████████▋                        | 42/100 [00:42<00:59,  1.02s/it] 43%|██████████████████                        | 43/100 [00:43<00:58,  1.02s/it] 44%|██████████████████▍                       | 44/100 [00:44<00:56,  1.02s/it] 45%|██████████████████▉                       | 45/100 [00:45<00:55,  1.01s/it] 46%|███████████████████▎                      | 46/100 [00:46<00:54,  1.01s/it] 47%|███████████████████▋                      | 47/100 [00:47<00:53,  1.01s/it] 48%|████████████████████▏                     | 48/100 [00:48<00:52,  1.00s/it] 49%|████████████████████▌                     | 49/100 [00:49<00:51,  1.01s/it] 50%|█████████████████████                     | 50/100 [00:50<00:50,  1.01s/it] 51%|█████████████████████▍                    | 51/100 [00:51<00:49,  1.00s/it] 52%|█████████████████████▊                    | 52/100 [00:52<00:48,  1.00s/it] 53%|██████████████████████▎                   | 53/100 [00:52<00:46,  1.00it/s] 54%|██████████████████████▋                   | 54/100 [00:54<00:46,  1.01s/it] 55%|███████████████████████                   | 55/100 [00:55<00:45,  1.00s/it] 56%|███████████████████████▌                  | 56/100 [00:55<00:43,  1.00it/s] 57%|███████████████████████▉                  | 57/100 [00:57<00:43,  1.00s/it] 58%|████████████████████████▎                 | 58/100 [00:57<00:41,  1.00it/s] 59%|████████████████████████▊                 | 59/100 [00:58<00:40,  1.00it/s] 60%|█████████████████████████▏                | 60/100 [00:59<00:39,  1.00it/s] 61%|█████████████████████████▌                | 61/100 [01:00<00:38,  1.01it/s] 62%|██████████████████████████                | 62/100 [01:01<00:37,  1.01it/s] 63%|██████████████████████████▍               | 63/100 [01:02<00:36,  1.01it/s] 64%|██████████████████████████▉               | 64/100 [01:03<00:35,  1.01it/s] 65%|███████████████████████████▎              | 65/100 [01:04<00:34,  1.01it/s] 66%|███████████████████████████▋              | 66/100 [01:05<00:33,  1.01it/s] 67%|████████████████████████████▏             | 67/100 [01:06<00:32,  1.01it/s] 68%|████████████████████████████▌             | 68/100 [01:07<00:31,  1.01it/s] 69%|████████████████████████████▉             | 69/100 [01:08<00:30,  1.01it/s] 70%|█████████████████████████████▍            | 70/100 [01:09<00:29,  1.01it/s] 71%|█████████████████████████████▊            | 71/100 [01:10<00:28,  1.01it/s] 72%|██████████████████████████████▏           | 72/100 [01:11<00:27,  1.01it/s] 73%|██████████████████████████████▋           | 73/100 [01:12<00:26,  1.01it/s] 74%|███████████████████████████████           | 74/100 [01:13<00:25,  1.01it/s] 75%|███████████████████████████████▌          | 75/100 [01:14<00:24,  1.01it/s] 76%|███████████████████████████████▉          | 76/100 [01:15<00:23,  1.01it/s] 77%|████████████████████████████████▎         | 77/100 [01:16<00:22,  1.01it/s] 78%|████████████████████████████████▊         | 78/100 [01:17<00:21,  1.01it/s] 79%|█████████████████████████████████▏        | 79/100 [01:17<00:20,  1.02it/s] 80%|█████████████████████████████████▌        | 80/100 [01:18<00:19,  1.02it/s] 81%|██████████████████████████████████        | 81/100 [01:19<00:18,  1.02it/s] 82%|██████████████████████████████████▍       | 82/100 [01:20<00:17,  1.02it/s] 83%|██████████████████████████████████▊       | 83/100 [01:21<00:16,  1.02it/s] 84%|███████████████████████████████████▎      | 84/100 [01:22<00:15,  1.02it/s] 85%|███████████████████████████████████▋      | 85/100 [01:23<00:14,  1.01it/s] 86%|████████████████████████████████████      | 86/100 [01:24<00:13,  1.01it/s] 87%|████████████████████████████████████▌     | 87/100 [01:25<00:12,  1.01it/s] 88%|████████████████████████████████████▉     | 88/100 [01:26<00:11,  1.01it/s] 89%|█████████████████████████████████████▍    | 89/100 [01:27<00:10,  1.01it/s] 90%|█████████████████████████████████████▊    | 90/100 [01:28<00:09,  1.02it/s] 91%|██████████████████████████████████████▏   | 91/100 [01:30<00:08,  1.01it/s] 92%|██████████████████████████████████████▋   | 92/100 [01:31<00:07,  1.01it/s] 93%|███████████████████████████████████████   | 93/100 [01:32<00:06,  1.01it/s] 94%|███████████████████████████████████████▍  | 94/100 [01:32<00:05,  1.01it/s] 95%|███████████████████████████████████████▉  | 95/100 [01:34<00:04,  1.01it/s] 96%|████████████████████████████████████████▎ | 96/100 [01:35<00:03,  1.01it/s] 97%|████████████████████████████████████████▋ | 97/100 [01:35<00:02,  1.01it/s] 98%|█████████████████████████████████████████▏| 98/100 [01:36<00:01,  1.01it/s] 99%|█████████████████████████████████████████▌| 99/100 [01:37<00:00,  1.01it/s]100%|█████████████████████████████████████████| 100/100 [01:38<00:00,  1.01it/s]
  0%|                                                   | 0/109 [00:00<?, ?it/s]  1%|▍                                          | 1/109 [00:01<03:11,  1.77s/it]  2%|▊                                          | 2/109 [00:02<02:36,  1.47s/it]  3%|█▏                                         | 3/109 [00:04<02:24,  1.36s/it]  4%|█▌                                         | 4/109 [00:05<02:18,  1.32s/it]  5%|█▉                                         | 5/109 [00:06<02:13,  1.28s/it]  6%|██▎                                        | 6/109 [00:07<02:08,  1.25s/it]  6%|██▊                                        | 7/109 [00:08<02:06,  1.24s/it]  7%|███▏                                       | 8/109 [00:09<02:05,  1.24s/it]  8%|███▌                                       | 9/109 [00:10<02:01,  1.22s/it]  9%|███▊                                      | 10/109 [00:12<02:00,  1.22s/it] 10%|████▏                                     | 11/109 [00:13<01:59,  1.21s/it] 11%|████▌                                     | 12/109 [00:14<01:56,  1.20s/it] 12%|█████                                     | 13/109 [00:15<01:54,  1.19s/it] 13%|█████▍                                    | 14/109 [00:16<01:53,  1.19s/it] 14%|█████▊                                    | 15/109 [00:17<01:50,  1.17s/it] 15%|██████▏                                   | 16/109 [00:18<01:47,  1.16s/it] 16%|██████▌                                   | 17/109 [00:19<01:46,  1.16s/it] 17%|██████▉                                   | 18/109 [00:20<01:45,  1.16s/it] 17%|███████▎                                  | 19/109 [00:22<01:44,  1.17s/it] 18%|███████▋                                  | 20/109 [00:22<01:42,  1.15s/it] 19%|████████                                  | 21/109 [00:24<01:41,  1.15s/it] 20%|████████▍                                 | 22/109 [00:25<01:40,  1.15s/it] 21%|████████▊                                 | 23/109 [00:26<01:39,  1.15s/it] 22%|█████████▏                                | 24/109 [00:27<01:37,  1.15s/it] 23%|█████████▋                                | 25/109 [00:28<01:36,  1.14s/it] 24%|██████████                                | 26/109 [00:29<01:35,  1.15s/it] 25%|██████████▍                               | 27/109 [00:30<01:34,  1.15s/it] 26%|██████████▊                               | 28/109 [00:32<01:33,  1.15s/it] 27%|███████████▏                              | 29/109 [00:33<01:32,  1.15s/it] 28%|███████████▌                              | 30/109 [00:34<01:31,  1.15s/it] 28%|███████████▉                              | 31/109 [00:35<01:29,  1.15s/it] 29%|████████████▎                             | 32/109 [00:36<01:28,  1.15s/it] 30%|████████████▋                             | 33/109 [00:38<01:27,  1.16s/it] 31%|█████████████                             | 34/109 [00:39<01:27,  1.16s/it] 32%|█████████████▍                            | 35/109 [00:40<01:26,  1.16s/it] 33%|█████████████▊                            | 36/109 [00:41<01:25,  1.16s/it] 34%|██████████████▎                           | 37/109 [00:42<01:23,  1.16s/it] 35%|██████████████▋                           | 38/109 [00:44<01:22,  1.16s/it] 36%|███████████████                           | 39/109 [00:45<01:21,  1.16s/it] 37%|███████████████▍                          | 40/109 [00:46<01:20,  1.16s/it] 38%|███████████████▊                          | 41/109 [00:47<01:18,  1.16s/it] 39%|████████████████▏                         | 42/109 [00:48<01:17,  1.16s/it] 39%|████████████████▌                         | 43/109 [00:50<01:16,  1.16s/it] 40%|████████████████▉                         | 44/109 [00:51<01:15,  1.17s/it] 41%|█████████████████▎                        | 45/109 [00:52<01:14,  1.16s/it] 42%|█████████████████▋                        | 46/109 [00:53<01:13,  1.16s/it] 43%|██████████████████                        | 47/109 [00:54<01:12,  1.17s/it] 44%|██████████████████▍                       | 48/109 [00:56<01:11,  1.17s/it] 45%|██████████████████▉                       | 49/109 [00:57<01:10,  1.17s/it] 46%|███████████████████▎                      | 50/109 [00:58<01:09,  1.17s/it] 47%|███████████████████▋                      | 51/109 [00:59<01:07,  1.17s/it] 48%|████████████████████                      | 52/109 [01:00<01:06,  1.17s/it] 49%|████████████████████▍                     | 53/109 [01:01<01:05,  1.17s/it] 50%|████████████████████▊                     | 54/109 [01:02<01:04,  1.17s/it] 50%|█████████████████████▏                    | 55/109 [01:04<01:03,  1.17s/it] 51%|█████████████████████▌                    | 56/109 [01:05<01:01,  1.17s/it] 52%|█████████████████████▉                    | 57/109 [01:06<01:00,  1.17s/it] 53%|██████████████████████▎                   | 58/109 [01:07<00:59,  1.17s/it] 54%|██████████████████████▋                   | 59/109 [01:08<00:58,  1.16s/it] 55%|███████████████████████                   | 60/109 [01:09<00:56,  1.16s/it] 56%|███████████████████████▌                  | 61/109 [01:10<00:55,  1.16s/it] 57%|███████████████████████▉                  | 62/109 [01:12<00:54,  1.16s/it] 58%|████████████████████████▎                 | 63/109 [01:13<00:53,  1.16s/it] 59%|████████████████████████▋                 | 64/109 [01:14<00:52,  1.16s/it] 60%|█████████████████████████                 | 65/109 [01:15<00:51,  1.16s/it] 61%|█████████████████████████▍                | 66/109 [01:16<00:50,  1.16s/it] 61%|█████████████████████████▊                | 67/109 [01:18<00:48,  1.17s/it] 62%|██████████████████████████▏               | 68/109 [01:19<00:47,  1.17s/it] 63%|██████████████████████████▌               | 69/109 [01:20<00:46,  1.17s/it] 64%|██████████████████████████▉               | 70/109 [01:21<00:45,  1.17s/it] 65%|███████████████████████████▎              | 71/109 [01:22<00:44,  1.17s/it] 66%|███████████████████████████▋              | 72/109 [01:24<00:43,  1.17s/it] 67%|████████████████████████████▏             | 73/109 [01:25<00:42,  1.17s/it] 68%|████████████████████████████▌             | 74/109 [01:26<00:40,  1.17s/it] 69%|████████████████████████████▉             | 75/109 [01:27<00:39,  1.16s/it] 70%|█████████████████████████████▎            | 76/109 [01:28<00:38,  1.16s/it] 71%|█████████████████████████████▋            | 77/109 [01:29<00:37,  1.16s/it] 72%|██████████████████████████████            | 78/109 [01:30<00:36,  1.16s/it] 72%|██████████████████████████████▍           | 79/109 [01:31<00:34,  1.16s/it] 73%|██████████████████████████████▊           | 80/109 [01:33<00:33,  1.16s/it] 74%|███████████████████████████████▏          | 81/109 [01:34<00:32,  1.16s/it] 75%|███████████████████████████████▌          | 82/109 [01:35<00:31,  1.16s/it] 76%|███████████████████████████████▉          | 83/109 [01:36<00:30,  1.16s/it] 77%|████████████████████████████████▎         | 84/109 [01:38<00:29,  1.17s/it] 78%|████████████████████████████████▊         | 85/109 [01:39<00:28,  1.17s/it] 79%|█████████████████████████████████▏        | 86/109 [01:40<00:26,  1.17s/it] 80%|█████████████████████████████████▌        | 87/109 [01:41<00:25,  1.17s/it] 81%|█████████████████████████████████▉        | 88/109 [01:42<00:24,  1.17s/it] 82%|██████████████████████████████████▎       | 89/109 [01:44<00:23,  1.17s/it] 83%|██████████████████████████████████▋       | 90/109 [01:45<00:22,  1.17s/it] 83%|███████████████████████████████████       | 91/109 [01:46<00:21,  1.17s/it] 84%|███████████████████████████████████▍      | 92/109 [01:47<00:19,  1.17s/it] 85%|███████████████████████████████████▊      | 93/109 [01:48<00:18,  1.17s/it] 86%|████████████████████████████████████▏     | 94/109 [01:49<00:17,  1.17s/it] 87%|████████████████████████████████████▌     | 95/109 [01:50<00:16,  1.17s/it] 88%|████████████████████████████████████▉     | 96/109 [01:51<00:15,  1.16s/it] 89%|█████████████████████████████████████▍    | 97/109 [01:52<00:13,  1.16s/it] 90%|█████████████████████████████████████▊    | 98/109 [01:54<00:12,  1.16s/it] 91%|██████████████████████████████████████▏   | 99/109 [01:55<00:11,  1.16s/it] 92%|█████████████████████████████████████▌   | 100/109 [01:56<00:10,  1.16s/it] 93%|█████████████████████████████████████▉   | 101/109 [01:57<00:09,  1.16s/it] 94%|██████████████████████████████████████▎  | 102/109 [01:58<00:08,  1.16s/it] 94%|██████████████████████████████████████▋  | 103/109 [01:59<00:06,  1.16s/it] 95%|███████████████████████████████████████  | 104/109 [02:00<00:05,  1.16s/it] 96%|███████████████████████████████████████▍ | 105/109 [02:02<00:04,  1.16s/it] 97%|███████████████████████████████████████▊ | 106/109 [02:03<00:03,  1.16s/it] 98%|████████████████████████████████████████▏| 107/109 [02:04<00:02,  1.16s/it] 99%|████████████████████████████████████████▌| 108/109 [02:05<00:01,  1.17s/it]100%|█████████████████████████████████████████| 109/109 [02:07<00:00,  1.17s/it]
