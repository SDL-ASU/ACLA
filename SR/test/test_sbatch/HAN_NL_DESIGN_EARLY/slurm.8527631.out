Making model...
Loading model from /data/yyang409/yancheng/SR_results_torch16/HAN_NL_DESIGN_EARLY_x2/HAN_NL_DESIGN_EARLY_scratch/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 2.82s, ave time: 0.56s

Making model...
Loading model from /data/yyang409/yancheng/SR_results_torch16/HAN_NL_DESIGN_EARLY_x2/HAN_NL_DESIGN_EARLY_scratch/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 8.10s, ave time: 0.58s

Making model...
Loading model from /data/yyang409/yancheng/SR_results_torch16/HAN_NL_DESIGN_EARLY_x2/HAN_NL_DESIGN_EARLY_scratch/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 49.96s, ave time: 0.50s

Making model...
Loading model from /data/yyang409/yancheng/SR_results_torch16/HAN_NL_DESIGN_EARLY_x2/HAN_NL_DESIGN_EARLY_scratch/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 99.03s, ave time: 0.99s

Making model...
Loading model from /data/yyang409/yancheng/SR_results_torch16/HAN_NL_DESIGN_EARLY_x2/HAN_NL_DESIGN_EARLY_scratch/model/model_best.pt

Evaluation:
[MyImage x2]	PSNR: 0.000 (Best: 0.000 @epoch 1)
Total time: 127.17s, ave time: 1.17s

